app/src/main/java/org/liberty/android/fantastischmemo/common/AnyMemoDBOpenHelper.java

```diff
@@ -199,11 +204,22 @@ public void onUpgrade(SQLiteDatabase database, ConnectionSource connectionSource
         if(oldVersion <= 7){
             database.execSQL(CREATE_MULTIPLE_CHOICE_TABLE);
         }
-
+      
+        final String CREATE_HISTORY_TABLE = "create table " +
```

Comment:
Move this String somewhere else, maybe in the if statement for if oldVersion <= 9